src/exif.c

```diff
@@ -142,6 +142,52 @@ avifResult avifImageExtractExifOrientationToIrotImir(avifImage * image)
     return AVIF_RESULT_OK;
 }
 
+uint8_t avifImageIrotImirToExifOrientation(const avifImage * image)
```

Comment:
Move this function to src/write.c as a static function, or put this function inside `#if defined(AVIF_ENABLE_EXPERIMENTAL_METAV1)`.